### PR TITLE
clang-format: Enable `SortIncludes`

### PR DESCRIPTION
**Pull request by glebm**
This fixes the clang-tidy warnings about the include order

clang-tidy can warn about these but not fix them automatically, while clang-format can fix them.

### CODE REVIEW

File: Source/controls/game_controls.cpp [hunk 1]
```diff
@@ -277,7 +277,7 @@ bool GetGameAction(const SDL_Event &event, ControllerButtonEvent ctrlEvent, Game
 					CalcViewportGeometry();
 				}
 #else
-			    // Not mapped. Reserved for future use.
+			                                // Not mapped. Reserved for future use.
 #endif
 				return true;
```
Comment (AJenbo) [inline Source/controls/game_controls.cpp:282]: ???

File: test/animationinfo_test.cpp [hunk 2]
```diff
@@ -219,8 +219,8 @@ TEST(AnimationInfo, AttackSwordWarriorWithFastestAttack) // Skipped frames and P
 }
 
 /*
```
Comment (AJenbo) [inline test/animationinfo_test.cpp:221]:
```suggestion
/**
```

File: test/animationinfo_test.cpp [hunk 3]
```diff
@@ -62,11 +62,11 @@ struct RenderingData : TestData {
 };
 
 /*
```
Comment (AJenbo) [inline test/animationinfo_test.cpp:64]:
```suggestion
/**
```

File: test/animationinfo_test.cpp [hunk 4]
```diff
@@ -50,8 +50,8 @@ struct GameTickData : TestData {
  */
 struct RenderingData : TestData {
 	/*
```
Comment (AJenbo) [inline test/animationinfo_test.cpp:52]:
```suggestion
	/**
```

File: Source/utils/sdl_cond.h [hunk 5]
```diff
@@ -1,48 +1,49 @@
...
-} // namespace devilution
+#pragma once
+
+#include <SDL.h>
+
+#include "sdl_mutex.h"
+
+namespace devilution {
+
+/*
```
Comment (AJenbo) [inline Source/utils/sdl_cond.h:9]:
```suggestion
/**
```

File: Source/utils/sdl2_to_1_2_backports.cpp [hunk 6]
```diff
@@ -834,12 +834,12 @@ char *SDL_GetPrefPath(const char *org, const char *app)
 {
 	// From sdl2-2.0.9/src/filesystem/unix/SDL_sysfilesystem.c
 	/*
```
Comment (AJenbo) [inline Source/utils/sdl2_to_1_2_backports.cpp:836]:
```suggestion
	/**
```

File: Source/controls/game_controls.cpp (hunk 1 above)
Comment (glebm) [reply Source/controls/game_controls.cpp]: No idea what confused clang-format here but changing the comment from `//` to `/* */` seems to have fixed it.

**Review by AJenbo — DISMISSED** [review]

**Comment by AJenbo** [comment]
Rebase needed

**Comment by glebm** [comment]
Rebased

**Review by AJenbo — APPROVED** [review]
Nice to have all the comments fixed :+1:

**Pull request merged**